content-src/lib/filters.js

```diff
@@ -0,0 +1,28 @@
+function createFilter(definition) {
+  return function(item) {
+    let result = true;
+    definition.forEach(test => {
+      if (!test(item)) {
+        result = false;
+      }
+    });
+    return result;
+  };
+}
+
+const URL_FILTERS = [
+  item => !!item.url,
+  item => item.url && item.url.length < 100,
```

Comment:
Why an arbitrary URL length limit?

For example, why wouldn't I want to see URLs like: `https://github.com/k88hudson/activity-streams/commit/eceda0e47e9f4a810bddf089e08c7b04dea1688f` (don't actually fact check the length, it's only like 94 chars, but you get the point).

Also, bonus points for making the 100 magic number a `const` at the top of the file, like `MAX_URL_LEN` or whatever.